{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/settings/dist.py

```diff
@@ -86,4 +86,11 @@
     }
 }
 
+MIDDLEWARE = [
+    'django.middleware.security.SecurityMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```

Comment:
I'd appreciate if whitenoise was flag-controlled, there will be deployments where it's undesirable.

Reply:
I would be against the flag for whitenoise: since it is deployment thing, I would recommend just add it commented (and in the `requirements` as well). The flag adds additional mechanism which will be used just once in the product lifecycle.

Reply:
What about a cookiecutter flag during project build? Cookiecutter allows conditional command line parameters to customize the final build.

Reply:
The question is: in how many projects we will know in advance where and how the code will be deployed? Adding `whitenoise` to the project is a job for few minute. So I'd rather have a comment somewhere in config how to add it quickly if necessary and ditch the cookiecutter flag (or any other flag) which should be documented etc.

Reply:
Agreed

Reply:
Hmmm, I think that they changed slightly the way reactions are displayed below the comments - and I think that I don't like it.